projects/vdk-core/tests/functional/run/jobs/ingest_to_two_dbs_with_different_types/ingest_step.py

```diff
@@ -0,0 +1,47 @@
+# Copyright 2023-2024 Broadcom
+# SPDX-License-Identifier: Apache-2.0
+import os
+
+from vdk.api.job_input import IJobInput
+
+
+def run(job_input: IJobInput):
+    duck_db_payload = {
+        "date": "2021-01-01",
+        "symbol": "BOOB",
+        "price": 123.0,
+    }
+
+    job_input.send_object_for_ingestion(
+        payload=duck_db_payload,
+        destination_table="test_duckdb_table",
+        method="duckdb",
+        target=os.getenv("DUCKDB_DATABASE"),
```

Comment:
Does this work? 
We support the target already?

Reply:
@murphp15 yes, it works with different dbs fro different types. If the dbs are from. the same type it won't work well since the conf variables are per plugin, not per db